components/draw/AttributeMode.svelte

```diff
@@ -0,0 +1,91 @@
+<script>
+  import geojsonExtent from "@mapbox/geojson-extent";
+  import {
+    map,
+    gjScheme,
+    currentHover,
+    currentlyEditing,
+    setCurrentlyEditing,
+    openFromSidebar,
+  } from "../../stores.js";
+
+  export let mode;
+  export let drawControls;
+  // Maybe null at first
+  export let routeSnapper;
+
+  $: {
+    if (mode == "edit-attribute") {
+      drawControls.changeMode("static");
+      // TODO The stop() implementation should be robust to the source not being ready yet
```

Comment:
General frustration: I can't currently wire up local ATIP to local route snapper edits. So small problems like this require me to fix the problem elsewhere and publish a new NPM version. Still trying to find a better workflow.